cmd/root.go

```diff
@@ -21,6 +22,16 @@ func NewRootCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "models",
 		Short: "GitHub Models extension",
+		Long: heredoc.Docf(`
+			GitHub Models CLI extension allows you to experiment with AI models from the command line.
+
+			To see a list of all available commands, run %[1]sgh models help%[1]s. To run the extension in
+			interactive mode, run %[1]sgh models run%[1]s. This will prompt you to select a model and then
+			to enter a prompt. The extension will then return a response from the model.
+
+			For more information about what you can do with GitHub Models extension see the manual
+			at https://cli.github.com/manual/gh_models
```

Comment:
Guessing this URL is aspirational. It currently 404s; wondering if we should omit the URL from the output till the page exists. Could use the readme URL in the meantime: https://github.com/github/gh-models/blob/main/README.md.